floodviz/utils.py

```diff
@@ -0,0 +1,90 @@
+import requests
+
+"""
+    Takes a valid url and a dictionary of expected column header strings as keys (values do not matter).
+    parse_rdb then requests data from the service at url, and extracts data matching any header values.
+     
+    Data is returned as a list of dict objects, where each object represents a data point. 
+
+"""
+
+
+def parse_rdb(url, header_values):
+    content = None
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        print('- Bad URL -')
+    else:
+        if r.status_code is 200:
+            content = r.text.splitlines()
+    if content:
+        all_data = []
+        keep_headers = {}
```

Comment:
I don't see why you avoid using `header_values` to store the positions of the headers. It's fine to use a new dictionary, but this way there's no reason for `header_values` to be a dictionary because only its keys are ever used.